_dev/src/components/product-feed-page/sync-overview.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <b-card
+    body-class="p-0"
+  >
+    <template #header>
+      <h1 class="mb-0 ps_gs-onboardingcard__title">
+        {{ $t('productFeedSettings.breadcrumb1') }}
+      </h1>
+    </template>
+    <div class="d-sm-flex">
+      <div class="flex-shrink-0 d-flex flex-column p-3">
+        <SyncHistory
+          v-slot="{ syncStates }"
+        >
+          <SyncState
+            v-for="syncState, i in syncStates"
+            :key="i"
+            v-bind="{...syncState}"
```

Comment:
not sure about this way of passing props as a destructured object, might need to check with @Quetzacoalt91  ?

Reply:
Would you suggest it to be more explicit ? 

```
:title="syncState.title"
:description="syncState.description"
:icon="syncState.icon"
lineColor="syncState.lineColor"
```

Or to pass an object and change `syncState` component to expect only one props `syncState` ?

Reply:
No for me it's just 
`:syncState`
I don't understand why we would need to write `{...syncState}`

Reply:
`v-bind="{...syncState}"` is equivalent to:
```
:title="syncState.title"
:description="syncState.description"
:icon="syncState.icon"
:lineColor="syncState.lineColor"
```

`:syncState="syncState"` is equivalent to:
```
:syncState="syncState"
```

In the first case, the destructuration allows to pass multiple props at once, 
In the second case, you pass only one props.

I went with the first solution to be able to add a validator for each props, but if it's not a best practice, I'm eager to change it.

Reply:
You were correct before, I can do: `v-bind="syncState`.

About the validator, you're right, I should be able to validate the object.